Tests/Api/Report/GeoLocationPerformanceReportTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+
+namespace Tests\Werkspot\BingAdsApiBundle\Tests\Api\Report;
+
+use BingAds\Reporting\AccountThroughAdGroupReportScope;
+use BingAds\Reporting\GeoLocationPerformanceReportRequest;
+use BingAds\Reporting\NonHourlyReportAggregation;
+use BingAds\Reporting\ReportFormat;
+use BingAds\Reporting\ReportTime;
+use Werkspot\BingAdsApiBundle\Api\Report\GeoLocationPerformanceReport;
+
+class GeoLocationPerformanceReportTest extends \PHPUnit_Framework_TestCase
+{
+    const YESTERDAY = 'Yesterday';
+
+    public function testGetRequest()
+    {
+        $expected = new GeoLocationPerformanceReportRequest();
```

Comment:
Is any of this stuff required? If so we should move it to a constructor or factory method